src/seedu/addressbook/data/person/PostalCode.java

```diff
@@ -0,0 +1,29 @@
+package seedu.addressbook.data.person;
+
+public class PostalCode{
+	
+	private String postalCodeNum;
```

Comment:
May be it's better if you can use a generic variable name here such as, `value`.